in-depth/submission_guidelines.md

```diff
@@ -0,0 +1,177 @@
+
+### Thumbnail
+
+The absolute location of your thumbnail image.
+
+### Categories
+
+These categories are displayed along the top of the page.  By clicking them, you can filter the results to just those of the category.  Each category below has a special short-name to use in your YAML.  We may add more categories in the future (let us know if you have any suggestions).
+
+#### Available Categories:
+
+##### Themes
+* Single Page (`single-page`) - A single page website, where all content is displayed on the `index` page.
+* Multiple Pages (`multi-page`) - A theme suitable for multiple pages, the opposite of single-page. (This category is kind of redundant isn't it.  It may not survive until the final version...)
+* Personal Website (`personal`) - A theme intended to be used as a portfolio of your work or to establish an online presence on your personal domain. (Maybe this would be better refered to as `portfolio`, as most websites you'd make yourself could be considered "personal")
+* Blog-Style (`blog`) - A theme styled with blogging or news posts in mind.
+
+##### Plugins
+* Administration (`admin`) - A plugin that adds administration capabilities to Pico.
+* Developer-Focused (`developer`) - A plugin that adds developer-focused functionality. (Not really a good category...)
+* Theming Extensions (`theming`) - A plugin that adds extends twig by adding extra variables or functionality.
+
+### Meta
+
+A short string of descriptive text that displays below your heading when you mouse over your item.
+
+### Link
+
+The URL of the "download" link, at the bottom of your description.
+
+### Images
+
+These are your `fullsize` images.  Specify the absolute location of each image.
+
+### Info
+
+Here you can specify a few lines of "info" that will display above your description.  The recommended items are `By` and `Last Updated`, but you can create your own items if they'd better fit the context of your theme or plugin.
+
+If your entry is a port of someone else's work, please use the `Ported By` and `Original By` lines instead of the regular `By`.  We'd like to make sure that the original content creators receive proper credit for their work.
+
+Also, as visible in the template below, you can add links to these `info` items.  You can easily link your `By` line to your GitHub profile or another location.  If you're using the `Original By` line, you should also link to the original source in this manner.  You can use either HTML or Markdown links, but if you use Markdown, you'll want to wrap the entire string in quotes.  This is because the `[]` characters have a special meaning in YAML and must be escaped.
+
+You can also use `_blank:` to create a line that doesn't start with an identifier.
+
+### Description
+
+Finally, your description should go after the YAML, in the Markdown body.  Your description can be as long as you'd like, but keep in mind that the description area is very narrow (it's roughly 40 characters wide).
```

Comment:
> keep in mind that the description area is very narrow (it's roughly 40 characters wide)

That's only true for themes (`jumbovision` layout). The `card` layout (i.e. plugins) even require a longer description, otherwise the detail view looks very odd.